### PR TITLE
chore: add new maintainers to README 🚀

### PR DESCRIPTION
**Pull request by bajtos**
Add @derdeka, @dougal83 and @achrinza to the list of maintainers.

It looks like GitHub is not able to down-scale the auto-generated avatars :(

Screen Shot 2019-12-13 at 15 24 54

@derdeka @dougal83 Would you mind updating your profile to set your own avatar? It does not have to be your real picture, any image will do.

I will understand if you don't want to, in which case I am proposing to leave out the auto-generated avatars from the table in the README.

@dhmlau I would like to wait for  @derdeka, @dougal83 and @achrinza to approve this pull request before it's landed. To avoid waiting until I get back from my vacation, can you please take care of this pull request for me?

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- API Documentation in code was updated
- Documentation in [/docs/site](../tree/master/docs/site) was updated
- Affected artifact templates in `packages/cli` were updated
- Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by dougal83** [comment]
I kinda liked my space invaders mushroom thing...  Updated pic /w name.

**Comment by bajtos** [comment]
@dougal83 
> Updated pic /w name.

Thank you!

**Comment by bajtos** [comment]
I think we are good now.

Screen Shot 2019-12-13 at 16 46 23

**Comment by dhmlau** [comment]
@dougal83, may I get an approval from you as well? Thanks!

File: README.md [hunk 1]
```diff
@@ -95,6 +95,8 @@ You can join the team by posting a comment to
 | [![nabdelgadir]](https://github.com/nabdelgadir) | [![marioestradarosa]](https://github.com/marioestradarosa) |        [![yaty]](https://github.com/yaty)        |
 |                 Dominique Emond                  |                         Agnes Lin                          |                 Deepak Rajamohan                 |
 |     [![emonddr]](https://github.com/emonddr)     |         [![agnes512]](https://github.com/agnes512)         | [![deepakrkris]](https://github.com/deepakrkris) |
+|                  Denny Bartelt                   |                      Doug "dougal83"                       |                  Rifa Achrinza                   |
```
Comment (dougal83) [inline README.md]: Please update name: `"Doug "dougal83"` to `Douglas McConnachie`

Comment (achrinza) [reply README.md]: @dougal83 It's been updated

**Comment by dhmlau** [comment]
It's merged! Thanks!

**Pull request merged**